src/tons_cli/job_group.py

```diff
@@ -21,6 +21,19 @@ def list_jobs():
             click.echo(job)
 
 
+@job.command('list-active')
+def list_jobs():
+    """List all active jobs"""
+    jobs = sdk().get_all_active_jobs()
+    if jobs is None:
+        click.echo("Failed to get active jobs", err=True)
+    else:
+        if len(jobs) == 0:
+            click.echo(f"No active jobs found")
+        for job in jobs:
+            click.echo(f"Job: {job.job_name} Status: [{job.status}] Start time: {job.start_time}")
+            click.echo(f"Latest APM: {job.latest_apm}")
```

Comment:
use [prettytable](https://ptable.readthedocs.io/en/latest/tutorial.html) here instead of printing line by line